test/src/main/java/org/vertexium/test/GraphTestBase.java

```diff
@@ -1685,6 +1685,42 @@ public void testMarkVertexAndPropertiesHidden() {
         assertEquals(30, v1.getPropertyValue("k2", "age"));
     }
 
+    @Test
+    public void testReindexHiddenProperties() {
```

Comment:
Can we move testUpdateVertexWithDeletedElasticsearchDocument to GraphTestBase as well? https://github.com/visallo/vertexium/blob/master/elasticsearch5/search-index/src/test/java/org/vertexium/elasticsearch5/Elasticsearch5SearchIndexTest.java#L238

Reply:
I took a quick look and I think for now those need to stay in ES specific tests due to this line: https://github.com/visallo/vertexium/blob/master/elasticsearch5/search-index/src/test/java/org/vertexium/elasticsearch5/Elasticsearch5SearchIndexTest.java#L239